src/libs/Navigation/Navigation.js

```diff
@@ -38,7 +38,7 @@ function navigate(route = ROUTES.HOME) {
     // If we're navigating to the signIn page, replace the existing route in the stack with the SignIn route so that we
     // don't mistakenly route back to any older routes after the user signs in
     if (route === ROUTES.SIGNIN) {
```

Comment:
Is it still possible to navigate to a `/signin` ? If we are removing the route then this code needs to be updated and also must be modified in the `linkingConfig`.

The flow you need to test is adding a secondary login while not authenticated and can use git blame to find the context behind this change.

<img width="1196" alt="Screen Shot 2021-04-21 at 6 23 19 AM" src="https://user-images.githubusercontent.com/32969087/115588265-456df800-a26a-11eb-97ae-d2a9e7272f86.png">

Reply:
I think so long as we update [this line](https://github.com/Expensify/Expensify.cash/blob/9efcc63e4eb9dd27fac35d7bfc92b3400554be0c/src/libs/actions/User.js#L144) to also redirect to `HOME` we should be fine and this can be updated to `ROUTES.SIGNIN`. I can make sure to test the second login validation flow when that's been updated.

Reply:
@marcaaron Gootcha, I was using git blame in my IDE and it doesn't show this information.